Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -55,6 +55,7 @@
 numWFIB.extend([prefixDet+96.0])   #CloseByPGun CE_E_Front_120um
 numWFIB.extend([prefixDet+100.0])  #CloseByPGun CE_H_Coarse_Scint
 numWFIB.extend([prefixDet+61.0])   #Nu Gun
+numWFIB.extend([prefixDet+151.85]) #Heavy ion reconstruction
```

Comment:
I would say let's keep it. We can send the production right for the GEN-SIM in the meantime not waiting for the next pre-release. Could you crosscheck the [sample](https://cms-pdmv-prod.web.cern.ch/relval/api/relvals/get_cmsdriver/CMSSW_14_1_0_pre5__Phase2_2026D110_noPU_HIN_GS-MinBias_14TeV-00001) (also @stahlleiton)?  To be compared with [29751.85](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-45bdef/40885/runTheMatrix-results/29751.85_HydjetQMinBias_5519GeV+2026D110_hi/cmdLog) 

Eventually, if this PR is not in an hurry, the production should end in the next days and we could integrate it by then (anyway before the next pre release). 

Note two things:
- while I could overwrite the GT string I would still prefer to run this in pre5 to keep the [same naming](https://github.com/cms-sw/cmssw/blob/1e818e702bdf5b4bd93a80e5a8edceb6638d7146/Configuration/PyReleaseValidation/python/relval_steps.py#L4300) to avoid confusion (in principle we could re-run the production and re-write the string there but seems a waste); 
- we can't use the `DBrealisticHLLHC` in pre5;
- the fragment has been copied by hand from https://github.com/cms-sw/cmssw/blob/d1b4049bd8dffaf205aa6e4f9c4851e06c50dc90/Configuration/Generator/python/Hydjet_Quenched_MinBias_5519GeV_cfi.py;

Reply:
@AdrianoDee : just had a look at the gen fragment you pointed and it looks fine to me. The only difference seems to be the aging customization and the beamspot .

Reply:
I have to correct myself, of course we can't run in pre5 given the `phase2_pp_on_AA` modifier doesn't exist there. So for the GEN SIM we need to wait pre7 and redo all the default samples. So, agreeing with @srimanob I would comment this. Unless we would have a failing DAS query for quite too long for me. We will submit the needed GEN-SIM as soon as pre7 is out.

Reply:
Commented out the line in 47ae27e7ab370495835788731b234817736f22a3

Reply:
Hi @AdrianoDee @stahlleiton @mandrenguyen 

Does GEN-SIM need a modifier? From reviewing your code, it seems to me that the modifier is not necessary for GEN-SIM, but it has been added because the line that adds it applies to all steps.

Reply:
It does not need a modifier in the gen-sim step at the moment since the modifier is only affecting the reco step. However I added it to all steps in case in the future we do add modifications that impact them.